instat/dlgOneWayFrequencies.vb

```diff
@@ -81,7 +75,6 @@ Public Class dlgOneWayFrequencies
         ucrPnlFrequencies.AddToLinkedControls(ucrChkFlip, {rdoGraph, rdoBoth}, bNewLinkedDisabledIfParameterMissing:=True, bNewLinkedAddRemoveParameter:=True)
 
         ucrNudGroups.SetParameter(New RParameter("auto.group", 9))
-        ucrNudGroups.SetParameterIncludeArgumentName(True)

```

Comment:
In the previous format, after setting on initialise to false it was setting weight.by and auto.group to be invisible, that is why i had to set it to `True` for both controls.